src/User/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { List, ListItem } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+
+export const User = () => {
+  const [user, setUser] = React.useState([]);
```

Comment:
you have to get userId from the url. in order to do that u can user hook from `react-router-dom` called `useParams`
so, add new line after this and type next code:
`const {id} = useParams();`